### ISSUE TITLE
Drop Python 3.6 / 3.7 support

### ISSUE THREAD

**Issue by KiLLuuuhh** [puzzle/puzzle.opnsense#95]
Since we introduced changes which use Python dataclasses, we encountered a compatibility issue with Python 3.6 since dataclasses were introduced in Python 3.7. Investigating the Python versions on our supported OPNsense versions showed, that our oldest supported version, OPNsense 22.7, already ships with Python 3.9, therefore I suggest dropping at least the end of life versions of python (3.6/3.7) in our collection. Ref: https://devguide.python.org/versions/

Output of OPNsense 22.7 instance:
```
vagrant@OPNsense:~ $ opnsense-version -O
{
    "product_abi": "22.7",
    "product_arch": "amd64",
    "product_copyright_owner": "Deciso B.V.",
    "product_copyright_url": "https://www.deciso.com/",
    "product_copyright_years": "2014-2022",
    "product_email": "project@opnsense.org",
    "product_flavour": "OpenSSL",
    "product_hash": "068ef7106",
    "product_id": "opnsense",
    "product_name": "OPNsense",
    "product_nickname": "Powerful Panther",
    "product_series": "22.7",
    "product_version": "22.7",
    "product_website": "https://opnsense.org/"
}

vagrant@OPNsense:~ $ /usr/local/bin/python3 -V
Python 3.9.13

vagrant@OPNsense:~ $ /usr/local/bin/python -V
Python 3.9.13
```

**Comment by DonGiovanni83** [comment]
I agree, those Python versions should be dropped 👍🏼

**Comment by DonGiovanni83** [comment]
Closed by #96
